core/commands/files/files.go

```diff
@@ -310,7 +310,7 @@ Examples:
 
 var FilesReadCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Read a file in a given mfs",
```

Comment:
this one is just wrong. should be:
`Read a file at a given path`.

And I'm really not on board with calling it unixfs... that describes the data format.